addons/mail/static/src/discuss/google_translate/common/message_patch.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Message" t-inherit-mode="extension">
+        <xpath expr="//t[@t-if='message.body']" position="replace">
+            <t t-if="message.showTranslation" t-out="message.translationValue"/>
+            <t t-elif="message.body" t-out="message.body"/>
+            <small class="fst-italic text-muted">
+                <p t-if="message.showTranslation">
+                    (Translated from: <t t-if="message.translationSource" t-out="message.translationSource"/>)
+                </p>
+                <p t-if="message.translationErrors">
+                    <i class="text-danger fa fa-warning" role="img" title="Translation Failure" aria-label="Translation Failure"/>
+                    (Translation Failure: <t t-if="message.translationErrors" t-out="message.translationErrors"/>)
+                </p>
+            </small>
+        </xpath>
+    </t>
+    <t t-inherit="mail.Message.actions" t-inherit-mode="extension">
+        <xpath expr="//Dropdown" position="before">
+            <button t-if="translatable()" class="btn px-1 py-0 rounded-0" t-att-class="message.showTranslation ? 'o-mail-Message-translated btn-link' : ''" t-on-click="onClickToggleTranslation" tabindex="0" t-att="{'title': message.showTranslation ? 'Revert' : 'Translate', 'aria-label': message.showTranslation ? 'Revert' : 'Translate'}"><i class="fa fa-lg fa-language"/></button>
```

Comment:
Don't use `t-att=`, prefer specific `t-att-title` or `t-att-aria-label` so the attributes are properly highlighted by code editor.

Also strings like `Translate` are not properly flagged as it is. You can define this in template above:
```xml
<t t-set="TRANSLATE">Translate</t>
<t t-set="REVERT">Revert</t>
<button t-att-title="message.showTranslation ? REVERT : TRANSLATE"/>
```

Reply:
Changed the code according to your comments.